js/windowsTitleBar/style.css

```diff
@@ -0,0 +1,99 @@
+#title-bar {
+    display: flex;
+    position: fixed;
+    background: #4a4a4a;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 32px;
+    padding-left: 10px;
+    justify-content: center;
+    align-items: center;
+    margin-left: -10px;
```

Comment:
Why use `margin-left` when you get full control over this element when you are making its position fixed?